demo/tree-view-demo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { TreeView } from '../src';
+import { TreeItemData } from '../src/components/tree-view/tree-view';
+
+interface TreeViewDemoState {
+    selectedItem: Object;
+}
+
+export const treeData: TreeItemData[] = [
```

Comment:
Right! Will have that in mind, make data meaningful and overall make the demo better like we discussed